src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts

```diff
@@ -180,7 +181,14 @@ export function transactionToActivity(
       additionalFields = parseMigrateCreateV3(info, chainId, tokens)
     }
 
-    return { ...defaultFields, ...additionalFields }
+    const activity = { ...defaultFields, ...additionalFields }
+
+    if (details.cancelled) {
+      activity.title = CancelledTransactionTitleTable[details.info.type]
+      activity.status = TransactionStatus.Confirmed
```

Comment:
Why not just make this a new `TransactionStatus.Cancelled`?

Reply:
TransactionStatus represents on-chain status, and there is no cancelled status, just a confirmed tx that overrides the original pending one